<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>693</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>811</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="menuDM" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonAddUnit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>93</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>AddUnit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStartCombat">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>287</y>
       <width>89</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>start Combat</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_editMap_menuToggle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Upravit Mapu</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonDM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DM Button</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetPlayer" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>81</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonEndTurn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>EndTurn</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMove">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Pohyb</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="box_startMenu" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>91</width>
      <height>291</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonMapLoad">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>89</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>LoadMap</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_newMap">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>new Map</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="box_newEmptyMapSetting" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>161</width>
      <height>441</height>
     </rect>
    </property>
    <widget class="QSlider" name="slider_newMap_X">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_newMap_X">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_newMap_X_ft">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_newMap_Y">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_newMap_Y_ft">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_newMap_create">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>vytvorit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_newMap_Y_m">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_newMap_X_m">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="slider_newMap_Y">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_newMap_back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>ZpÄ›t</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="box_editMap" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>81</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_editMap_end">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Konec Ãºprav</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
